python/aitubertalk/rooms/room_module.py

```
@@ -0,0 +1,357 @@
+Room management module for AITuberTalk SDK
+"""
+
+import asyncio
+from typing import List, Optional, TYPE_CHECKING
+from uuid import uuid4
+from datetime import datetime
+
+from ..core.exceptions import AITuberTalkError, ErrorCode
+from .types import (
+    Room, CreateRoomConfig, ParticipantConfig, JoinResult, 
+    Participant, RoomFilter, AITuberConfig
```

Ruff (0.11.9)

`.types.AITuberConfig` imported but unused

Remove unused import: `.types.AITuberConfig`

(F401)